docs/book/src/topics/workload-identity.md

```diff
@@ -0,0 +1,204 @@
+# Workload Identity
+
+Azure AD Workload identity is the next iteration of Azure AD Pod identity 
+that enables Kubernetes applications (e.g. CAPZ) to access Azure cloud 
+resources securely with Azure Active Directory.
+
+This document describes a quick start guide of using workload identity and 
+assumes that you have access to Azure cloud.
+
+Workload identity is currently worked upon and cloud provider azure 
+integration is in progress. Please refer to [this](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/3589) issue for details.
+For more information, please refer to the [proposal](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/docs/proposals/20221611-workload-identity-integration.md)
+
+## Workload Identity Quick Start Guide
+
+### Setting Up Management Cluster on Kind
+
+- Create a public and private key pair. For example, you can generate the 
+  key pairs using OpenSSL.
+
+  Generate a private key called `sa.key` using the following command:
+```bash
+$ openssl genrsa -out sa.key 2048
+```
+
+Set the environment variable `SERVICE_ACCOUNT_SIGNING_KEY_FILE` to the path of the
+generated `sa.key`. This ENV var will be used in the upcoming step. 
+Note: You can use `readlink -f sa.key` to get the absolute path of the key file.
+
+Generate a public key using the private key.
+```bash
+$ openssl rsa -in sa.key -pubout -out sa.pub
+```
+Set the environment variable `SERVICE_ACCOUNT_KEY_FILE` to the path of the
+generated `sa.pub`. This ENV var will be used in the upcoming step.
+
+- Create and upload Discovery and JWKS document using this [link](https://azure.github.io/azure-workload-identity/docs/installation/self-managed-clusters/oidc-issuer.html)
+
+- At this stage, you will need to create a federated identity credential.
+  - You can create that either with Azure AD application or user-assigned
+    identity. Please note that user assigned identity will need to be created
+    regardless because cloud provider azure integration is not yet done. The
+    steps are mentioned in the next section of workload cluster creation.
+  - The next list items links to steps on creating the federated
+    identity credential. You will need to set up several environment
+    variables:
+    - `SERVICE_ACCOUNT_NAMESPACE` : Namespace where the capz-manager pod
+      will run.
+    - `SERVICE_ACCOUNT_NAME` : Name of the capz-manager k8s service account.
```

Comment:
I maybe completely thick here by missing something obvious, can you please elaborate for my clarity on what does `SERVICE_ACCOUNT_NAME` implies? Or how can one find it?

Reply:
`SERVICE_ACCOUNT_NAME` is the name of the `ServiceAccount` that the capz pod will use. You can find its name in the config of the capz manager here. 
https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/config/rbac/service_account.yaml#L5
Though the actual name becomes `capz-manager` ( instead of `manager` ) after kustomize render.